test/regression.uts

```diff
@@ -1223,6 +1223,62 @@ for i in range(10):
 response = res[0][1]
 assert response[ICMP].type == 0
 
+= Test set of sent_time by sr
+~ netaccess IP ICMP
+packet = IP(dst="8.8.8.8")/ICMP()
+r = sr(packet)
+assert packet.sent_time is not None
```

Comment:
The logic between ˋsr` and `sr1` is the same. Because networking tests tend to fail more often, it would be good not to have too many doing the same thing :)

Reply:
Done